ubuntudesign/vanilla-framework
# First pass at building out Pattern lib with Metalsmith

This is a first pass at building the living pattern library which will live at vanillaframework.io

It's still very much a WIP but merging this into the `v1-dev` branch will allow me to go through in a more granular fashion and add issues for auditing each individual section.

This first step was to restructure the codebase to match conventions as agreed with the rest of the web team and move all current documentation at `/demo` into a sensible place in the new structure.

This will all need to be audited and refined before `v1`

## Done

- Restructured Sass into flat based file structure following ITCSS convention
- Added Metalsmith Pattern Lib, splitting each pattern into unique link
- Deleted /demo
- Added Browser support doc

## QA

Pull code and run `gulp develop` - a page at `http://localhost:3000` should open in a separate window.
